tests/scenario/test_start_statuses.py

```diff
@@ -6,27 +6,24 @@
 from typing import Type
 from unittest.mock import patch
 
-import k8s_charm
-import machine_charm
+import charm
 import pytest
 import yaml
 from ops import pebble
 from ops.testing import CharmType
-from scenario import Container, Context, ExecOutput, State, SubordinateRelation
+from scenario import Container, Context, ExecOutput, State
 
 CHARM_ROOT = Path(__file__).parent.parent.parent
 
 
-@pytest.fixture(params=["k8s", "lxd"])
+@pytest.fixture(params=["k8s"])
 def substrate(request):
     return request.param
 
 
 @pytest.fixture
 def charm_type(substrate) -> Type[CharmType]:
-    return {"lxd": machine_charm.GrafanaAgentMachineCharm, "k8s": k8s_charm.GrafanaAgentK8sCharm}[
-        substrate
-    ]
+    return {"k8s": charm.GrafanaAgentK8sCharm}[substrate]
```

Comment:
Feels like this entire function should probably be removed. Or at the very least rewritten as `return charm.GrafanaAgentK8sCharm`.

Reply:
#256